sphinx/themes/basic/static/searchtools.js

```diff
@@ -358,7 +369,7 @@ const Search = {
           const score = Math.round(100 * queryLower.length / entry.length);
           const result = [
             docNames[file],
-            titles[file],
+            _escapeHTML(titles[file]),
```

Comment:
Could we move this into the `_displayItem` function here?

https://github.com/sphinx-doc/sphinx/blob/23cd5d0565dc8d54fe100ccb07f6f25f961f2eb0/sphinx/themes/basic/static/searchtools.js#L93

(I _think_ that's the only place the value is used, but am checking..)

Reply:
Ah; `descr` can be built using it too:

https://github.com/sphinx-doc/sphinx/blob/23cd5d0565dc8d54fe100ccb07f6f25f961f2eb0/sphinx/themes/basic/static/searchtools.js#L474

Reply:
I think my preference would be to escape both of those within `_displayItem`; my thinking being that it's the placement of the contents into the HTML which could cause problems to arise, so escaping near there makes the intent clearer - e.g. any assignment to `innerHTML` has a corresponding `_escapeHTML`.

Reply:
Yeah that makes sense, I'll update.

Reply:
Thanks!  Re-reading the changeset now..